Modules/_decimal/libmpdec/mpdecimal.h

```diff
@@ -100,12 +100,12 @@ const char *mpd_version(void);
   #if defined(CONFIG_64) || defined(CONFIG_32)
     #error "cannot use CONFIG_64 or CONFIG_32 with UNIVERSAL."
   #endif
-  #if defined(__ppc__)
-    #define CONFIG_32
-    #define ANSI
-  #elif defined(__ppc64__)
+  #if defined(__ppc64__) || defined(__powerpc64__)
```

Comment:
[This](https://github.com/cpredef/predef/blob/master/Architectures.md#powerpc) is the list of possible macros.

I have experienced the problem with clang-16.